setup.py

```diff
@@ -27,5 +27,10 @@
     license='MIT',
     author='jarbasAI',
     author_email='jarbasai@mailfence.com',
-    description='Mesh Networking utilities for mycroft core'
+    description='Mesh Networking utilities for mycroft core',
+    entry_points={
+        'console_scripts': [
+            'HiveMind-server=jarbas_hive_mind.__main__:main'
```

Comment:
maybe a HiveMind-database script to handle registering/modifying/deleting clients would be a good next step? for now only with the functionality to add a client like above

Reply:
I was thinking on a web interface, but a script is necessary too.

Reply:
a web interface has been discussed previously but never got off the idea stage, if this is something you are interested in doing I will gladly review and recommend it in the wiki!